### PR TITLE
Details abstraction

### PR DESCRIPTION
**Pull request by Henni**
Abstracting the details view to allow easy modification.

TODO:
- [ ] template for address
- [ ] implement telephone item
- [ ] birthday element should be input date

### CODE REVIEW

**Comment by DeepDiver1975** [comment]
````
Uncaught Error: [$rootScope:infdig] 10 $digest() iterations reached. Aborting!

**Comment by Henni** [comment]
@DeepDiver1975 should be fixed

File: js/services/vCardProperties.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+		},
+		note: {
+			readableName: 'Note',
+			template: 'textarea'
+		},
+		tel: {
+			readableName: 'Telephone',
+			template: 'tel'
+		}
+	};
```
Comment (Henni) [inline js/services/vCardProperties.js]: @DeepDiver1975 This is what I'm aiming for.

Comment (DeepDiver1975) [inline js/services/vCardProperties.js]: nice :+1:

**Comment by DeepDiver1975** [comment]
I took the freedom to hack a bit on this :wink:

**Comment by DeepDiver1975** [comment]
we should hide all props we don't know - this happens in case a contacts is edited in evolution

bildschirmfoto von 2016-02-23 13-33-34

**Comment by DeepDiver1975** [comment]
@jancborchardt please have a look at the photo positioning in the details view - THX

bildschirmfoto von 2016-02-24 16-27-10

**Comment by jancborchardt** [comment]
Perfect! Let’s get this in so I can directly base the styling work on this.

**Comment by jancborchardt** [comment]
@DeepDiver1975 @Henni merge, so each of us can work on the individual TODOs left?

**Pull request merged**